src/components/30-organisms/modal/index.scss

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.o-modal {
+  display: none;
+  width: 100%;
+  height: 100%;
+
+  position: fixed;
+  // Highest z-index on the axa website
+  z-index: 3000;
+  left: 0;
+  top: 0;
+
+  background-color: rgba(0, 0, 0, 0.5);
+
+  @include breakpoint($mediaquery-xs-up) {
+    align-items: center;
+  }
+
+  &--open {
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+  }
+
+  &__upper-close-container {
+    display: flex;
+    z-index: 1;
+    margin: 10px 10px 0px;
+    box-shadow: 0px 1px 10px 0px rgba(0, 0, 0, 0.15);
+    justify-content: flex-end;
+    border-radius: 8px 8px 0px 0px;
+    background-color: white;
+
+    @include breakpoint($mediaquery-xs-up) {
+      width: 60%;
+      padding: 0px;
+      margin: 0px;
+      background-color: white;
+    }
+
+    &-button {
+      border: none;
+      background: transparent;
+      outline: none;
+      color: inherit;
+      font: inherit;
+      line-height: normal;
+      margin: 8px;
+      height: 40px;
+      padding: 8px;
+      
+      svg {
+        height: 24px;
+        width: 24px;
+      }
+
+      /* Remove excess padding and border in Firefox 4+ */
+      &::-moz-focus-inner {
```

Comment:
This rule can be removed. I tested it on firefox and it didn't change anything if we leave it out.